pyuvsim/tests/conftest.py

```diff
@@ -41,3 +42,9 @@ def setup_and_teardown_package():
     # clean up the test directory after
     if os.path.exists(testdir):
         shutil.rmtree(testdir)
+
+
+@pytest.fixture(autouse=True)
+def ignore_center_deprecation():
+    warnings.filterwarnings('ignore', message='The default for the `center` keyword has changed.',
+                            category=DeprecationWarning)
```

Comment:
I'm going to make an issue to remove this after the pyuvdata v1.5 branch is merged.